src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -197,8 +197,11 @@ export class ServiceFormComponent extends CdForm implements OnInit {
   ngOnInit(): void {
     this.action = this.actionLabels.CREATE;
     this.cephServiceService.getKnownTypes().subscribe((resp: Array<string>) => {
-      // Remove service type 'osd', this is deployed a different way.
-      this.serviceTypes = _.difference(resp, ['osd']).sort();
+      // Remove service types:
+      // osd       - This is deployed a different way.
+      // container - This is meant to be used for installing additional
+      //             services while bootstrapping the system only.
+      this.serviceTypes = _.difference(resp, ['container', 'osd']).sort();
```

Comment:
So the question is:

* either we allow raw access to `service_type: container`, or
* we hide `service_type: container` here and instead build a UI for specific use cases, like SNMP.